sample/guest01.ini

```diff
@@ -51,6 +51,7 @@ data_dir=/home/gvt/caas
 [aaf]
 path=/home/gvt/caas/scripts/aaf
 support_suspend=enable
+audio_type=hda
```

Comment:
could you also add a comment here to indicate there are also another option: sof. And explain the different between hda and sof briefly.